src/xdocs/writingchecks.xml

```diff
@@ -64,17 +64,17 @@
 
       <p>
         Suppose you have a convention that the number of methods in a
-        class should not exceed a certain limit, say 30. This rule makes
-        sense, a class should only do one thing and do it well. With a
+        class should not exceed 30. This rule makes
+        sense: a class should only do one thing and do it well. With a
         zillion methods chances are that the class does more than one
         thing. The only problem you have is that your convention is not
-        checked by Checkstyle, so you&#39;ll have to write your own Check
+        checked by Checkstyle. You&#39;ll have to write your own Check
         and plug it into the Checkstyle framework.
       </p>
 
       <p>
         This chapter is organized as a tour that takes you
-        through the process step by step and explains both the theoretical
+        through this process step by step and explains both the theoretical
```

Comment:
I actually think the original sentence makes more sense. We have not really specified what "this" refers to in this paragraph and are actually talking about what is coming up.

Reply:
Changed to "the process of writing a check"